emctl/cmd/client/command/apply/applier.go

```diff
@@ -90,24 +90,15 @@ func (mc *meshControllerApplier) Apply() error {
 	ctx, cancelFunc := context.WithTimeout(context.Background(), mc.timeout)
 	defer cancelFunc()
 	err := mc.client.V1Alpha1().MeshController().Create(ctx, mc.object)
-	for {
-		switch {
-		case err == nil:
-			return nil
-		case meshclient.IsConflictError(err):
-			err = mc.client.V1Alpha1().MeshController().Patch(ctx, mc.object)
-			if err != nil && meshclient.IsConflictError(err) {
```

Comment:
These code is unreachable, cause Patch wont return ConflictError forever.

Reply:
This is a retry-able mechanism to make sure the resource would be applied.  The function will be exited at the default branch of the switch.

Reply:
Thanks for your comments. Yes, it is a retry-able mechanism, but there are some logical errors in switch statement. 
The  condition in the if statement is always false at the case meshclient.IsConflictError and meshclient.IsNotFoundError of the switch, because the patch function wont return ConflictError and create function wont return NotFoundError forever.


 And, There is no need for a retry mechanism in the apply method. When an error occurs, the method can be exited directly, and the user can re-execute the apply command. In most cases the create function is executed successfully.

I think we need to refactor the implementation logic here, let's find an implementation that meets the needs and is concise. This PR is an implementation I gave, please take a look.